src/software/ai/hl/stp/play/play.cpp

```diff
@@ -310,6 +310,13 @@ Play::assignTactics(const WorldPtr &world_ptr, TacticVector tactic_vector,
             double robot_cost_for_tactic =
                 primitives.at(robot.id())->getEstimatedPrimitiveCost();
 
+            std::unordered_map<unsigned int, double> assignment_cost_overrides = tactic->robotAssignmentCostsOverride();
+            if (assignment_cost_overrides.contains(robot.id()))
+            {
+                // tune the cost of the prioritized robot so it gets selected more for this tactic
+                robot_cost_for_tactic *= assignment_cost_overrides.at(robot.id());
+            }
+
```

Comment:
Similarly to the other PR, I'm not sure if this is effective. If we scale the "cost" for all the tactics that a certain robot could take on, wouldn't that effectively make no difference in which tactic would get selected?